app/components/search/result-cards/document.hbs

```diff
@@ -1,27 +1,38 @@
-<AuCard @shadow={{true}} @size="small" as |c|>
+<AuCard @shadow={{true}} @size="small" class="result-card" as |c|>
   <c.header>
     <div class="au-u-flex au-u-flex--vertical-center au-u-flex--spaced-tiny">
       <p class="au-u-para-small au-u-muted au-u-medium">
-        {{t "created"}}
         {{date @created}}
       </p>
     </div>
-    <AuHeading @level="3" @skin="4">
+    <AuHeading @level="3" @skin="6">
       <AuLink
         class="au-u-padding-none"
-        @route="agenda.agendaitems.agendaitem"
+        @route="document"
         @model={{@documentId}}
+        @icon="document"
+        @iconAlignment="left"
       >
-        {{@title}}
+        {{@fileName}}
```

Comment:
Document card shows the file name, not the piece title. Is this intentional?

![image](https://user-images.githubusercontent.com/22411874/221194322-0e1eefbe-76bc-41f0-8c20-6cbf85ba4b2c.png)

Reply:
Let's take this up with Hannes at 16:00.
He was going to get back to us on what goes where in the document card, but unless I missed it, he never did, so I told Kobe to leave it as is for now. Small change if needed

Reply:
I would go for piece name as that is editable, file name is not.